app/build.gradle

```diff
@@ -29,15 +29,21 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+
+    buildFeatures {
+        viewBinding = true
+    }
 }
 
 dependencies {
 
-    implementation 'androidx.core:core-ktx:1.7.0'
+    implementation 'androidx.core:core-ktx:1.8.0'
     implementation 'androidx.appcompat:appcompat:1.4.2'
     implementation 'com.google.android.material:material:1.6.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+    implementation "androidx.fragment:fragment-ktx:1.5.0"
```

Comment:
Không cần thiết phải sử dụng lib này

Reply:
em định kiểu .commit{it = beginTransaction()} cho nó gọn hơn ạ

Reply:
Anh nhớ không nhầm thì k cần add lib kia thì vẫn dùng dc commit bình thường mà. Mới cả add cả 1 lib mà chỉ để làm điều này thì k nên